model/http/metrics.yaml

```diff
@@ -141,6 +162,9 @@ groups:
   - id: metric.http.client.open_connections
     type: metric
     metric_name: http.client.open_connections
+    annotations:
+      code_generation:
+        metric_value_type: int
```

Comment:
Does `int` cover all integer types? This uses `long` in the .NET implementation (not sure why we decided to do so).

Reply:
all ints are signed int64 in OTel